### DISCUSSION TITLE
Pass slugify options to gatsby-plugin-page-creator

### DISCUSSION THREAD

**Discussion by nagiek** [gatsbyjs/gatsby#28470] [Ideas / Feature Requests] [answered]
Hi,

I think we should be allowed to pass options to `gatsby-plugin-page-creator` to define the slug in the derived path. Presently we get the default option.

https://github.com/gatsbyjs/gatsby/blob/b6b5207979370c244e32977c87242064f067842c/packages/gatsby-plugin-page-creator/src/derive-path.ts#L82

Envisioned interface:
```
    {
      resolve: 'gatsby-plugin-page-creator',
      options: {
        slugifyOptions: {
          lowercase: false,
        },
      },
    },
```

Thanks

**Comment by nagiek** [comment]
Additionally, if no "page" argument is passed, a default of `/src/pages` is assumed?

**Reply by LekoArts** [reply]
No, path is a required argument. Gatsby itself only sets it like this: https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/bootstrap/load-plugins/load.ts#L236

**Comment by LekoArts** [comment, marked as answer]
I guess allowing the options to pass through makes sense. I'll put up a PR

**Reply by LekoArts** [reply]
PR is here: https://github.com/gatsbyjs/gatsby/pull/28485

**Reply by LekoArts** [reply]
PR is merged and will be released 2020/12/15 :)

**Reply by nagiek** [reply]
Thank you!
